scripts/bash/gitlab_startup.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Gitlab starup script
+# This script will either install, migrate or reconfigure Gitlab installation according to the values set in the instance metadata.
+# Installation / Migration will be executed once in a life of an instance.
+# Reconfiguration will be executed on every boot.
+# Reconfiguration updates the Gitlab's External URL and certificates if they changed through metadata.
+
+
+#Vars
+DEPLOYMENT_GCS_PREFIX=`curl -H "Metadata-Flavor: Google" http://169.254.169.254/computeMetadata/v1/instance/attributes/gcs-prefix`
+GITLAB_INSTALL_VERSION=`curl -H "Metadata-Flavor: Google" http://169.254.169.254/computeMetadata/v1/instance/attributes/gitlab-version`
+GCS_PATH_TO_BACKUP=`curl -H "Metadata-Flavor: Google" http://169.254.169.254/computeMetadata/v1/instance/attributes/gcs-path-to-backup`
+GCLOUD_LOG_NAME="gitlab-startup"
+BACKUP_ARCHIVE_PATH="/tmp/backup_archived.zip"
+
+#Imports
+gsutil cp $DEPLOYMENT_GCS_PREFIX/scripts/bash/gitlab_helpers.sh ./
+gsutil cp $DEPLOYMENT_GCS_PREFIX/scripts/bash/gcloud_logger.sh ./
+source ./gcloud_logger.sh
+source ./gitlab_helpers.sh
+
+
+# Start
+logger $GCLOUD_LOG_NAME "INFO" "Starting Gitlab instance setup"
+
+check_installation $GCLOUD_LOG_NAME
+set_gitlabVars $GCLOUD_LOG_NAME
+
+if [ $GITLAB_INSTALLED == 'false' ]; then
+    logger $GCLOUD_LOG_NAME "INFO" "Starting Gitlab installation"
+    gitlab_depsInstall $GCLOUD_LOG_NAME 
+    gitlab_install $GCLOUD_LOG_NAME $GITLAB_INSTALL_VERSION
+    setup_cicd_vars $GCLOUD_LOG_NAME
```

Comment:
Not consistent with function naming conventions.
Lower case and underscores: "this_is_example"

Reply:
My naming convention for variables is:
MY_VAR - For global bash variables
myVar - For local bash function variable

So I might change the function names but, it will be much more confusing in my opinion